### DISCUSSION TITLE
IRC API Like Twitch IRC API

### DISCUSSION THREAD

**Discussion by coinpusherworld** [streamlink/streamlink#4098] [Q&A]
Is there a IRC API script that can be used to add into a game we are looking to bring to BIGO.TV please Like twitch IRC < CLICK [HERE](https://dev.twitch.tv/docs/api/) Also [HERE](https://github.com/TwitchLib/TwitchLib) TO SEE WHAT I MEAN>

**Comment by bastimeyer** [comment]
Not sure what you're asking here and what this has to do with Streamlink.
